pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -69,7 +69,8 @@ stdenv.mkDerivation (finalAttrs: {
   checkFlags = [ "BCACHEFS_TEST_USE_VALGRIND=no" ];
 
   makeFlags = [
-    "PREFIX=${placeholder "out"}"
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
```

Comment:
What's the deal with this change?

Reply:
They changed the flags. Copied what the nix config of the repo does:

https://github.com/koverstreet/bcachefs-tools/blob/master/build.nix#L49


Won't build without.